### PR TITLE
Submit IanDCarroll.md to Community Directory

### PR DESCRIPTION
**Pull request by IanDCarroll**
#### Submit Profile for Community Initiative

copycats # 104.
#### Description

Adds one .md file to community directory
#### Context and Reason Why

Adding my profile to the ever-growing list of ones so that the initiative has some content to chew on and I become super-famous for being there when it all started! :taco:JS 
#### How Has This Been Tested?

I tested this by reading carefully, following the instructions, and refactoring when I noticed what was being asked of similar pull requests.
#### Credits

@IanDCarroll - solo

@gr2m - who made me aware that this existed.
@machikoyasuda - for teaching me to read.
@jdorfman - for teaching me how to commit. Also general awesomeness.

---

[Made at Fullstack LA](https://meetup.com/la-fullstack/events/234620974) :zap:

### CODE REVIEW

**Comment by machikoyasuda** [comment]
best PR message ever 💯

File: community/IanDCarroll.md [hunk 1]
```diff
@@ -0,0 +1,8 @@
+# Ian Carroll
+
+Ian has been learning the art of software development for about a year and has taken an outside-in agile-first approach. He is a certified ScrumMaster, loves Pair Programming and is currently learning TDD, XP, Lean and Kanban. He experiments with Agile at work, home, and on his journey learning software. He has a diverse background in Retail, Theatre, Chinese Philopsphy, Writing, Star Trek and Tea.
```
Comment (machikoyasuda) [inline community/IanDCarroll.md]: Let's go with 1st person, `I have been`

**Review by machikoyasuda — CHANGES_REQUESTED** [review]

File: community/IanDCarroll.md (hunk 1 above)
Comment (IanDCarroll) [reply community/IanDCarroll.md]: :+1: Okey-Dokey, Artichokie! :+1:

Comment (IanDCarroll) [reply community/IanDCarroll.md]: Oh, wait. I  see a typo...

File: community/IanDCarroll.md [hunk 2]
```diff
@@ -1,6 +1,6 @@
 # Ian Carroll
 
-Ian has been learning the art of software development for about a year and has taken an outside-in agile-first approach. He is a certified ScrumMaster, loves Pair Programming and is currently learning TDD, XP, Lean and Kanban. He experiments with Agile at work, home, and on his journey learning software. He has a diverse background in Retail, Theatre, Chinese Philopsphy, Writing, Star Trek and Tea.
+I have been learning the art of software development for about a year and have taken an outside-in agile-first approach. I'm also a certified ScrumMaster, love Pair Programming and am currently learning TDD, XP, Lean and Kanban. I experiment with Agile at work, home, and on my journey learning software. I have a diverse background in Retail, Theatre, Chinese Philopsphy, Writing, Star Trek and Tea.
```
Comment (machikoyasuda) [inline community/IanDCarroll.md]: `Philosophy` typo

**Review by machikoyasuda — CHANGES_REQUESTED** [review]

File: community/IanDCarroll.md [hunk 3]
```diff
@@ -0,0 +1,8 @@
+# Ian Carroll
+
+I have been learning the art of software development for about a year and have taken an outside-in agile-first approach. I'm also a certified ScrumMaster, love Pair Programming and am currently learning TDD, XP, Lean and Kanban. I experiment with Agile at work, home, and on my journey learning software. I have a diverse background in Retail, Theatre, Chinese Philopsphy, Writing, Star Trek and Tea.
```
Comment (machikoyasuda) [inline community/IanDCarroll.md]: Let's always spell out abbreviations on 1st reference, so: `Test Driven Development (TDD), Extreme Programming (XP)`

**Review by machikoyasuda — CHANGES_REQUESTED** [review]

File: community/IanDCarroll.md (hunk 3 above)
Comment (IanDCarroll) [reply community/IanDCarroll.md]: got it.

**Comment by machikoyasuda** [comment]
Squashing & merging! :100

**Comment by IanDCarroll** [comment]
Huzzah!

**Pull request merged**